### PR TITLE
Update (fav)icons

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Update (fav)icons as already processed in portal and II.

### CODE REVIEW

**Review by mstrasinskis — COMMENTED** [review]
LGTM, one suggestion.

File: frontend/static/assets/favicons/browserconfig.xml [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <browserconfig>
     <msapplication>
         <tile>
-            <square150x150logo src="/assets/mstile-150x150.png"/>
-            <TileColor>#383c3c</TileColor>
+            <square150x150logo src="/mstile-150x150.png"/>
```
[the remark this answers is not in the document]
Comment (mstrasinskis) [reply frontend/static/assets/favicons/browserconfig.xml]: Thank you!

[the remark this answers is not in the document]
Comment (peterpeterparker) [reply frontend/static/assets/favicons/browserconfig.xml]: Done 👍

**Review by mstrasinskis — APPROVED** [review]
LGTM

**Pull request merged**